pkg/reconciler/nscert/nscert.go

```diff
@@ -77,7 +78,8 @@ func (c *reconciler) ReconcileKind(ctx context.Context, ns *corev1.Namespace) pk
 		return fmt.Errorf("failed to list certificates: %w", err)
 	}
 
-	if ns.Labels[networking.DisableWildcardCertLabelKey] == "true" {
+	if strings.EqualFold(ns.Labels[networking.DisableWildcardCertLabelKey], "true") ||
```

Comment:
I agree with Victor's comment: https://github.com/knative/serving/pull/7530#issuecomment-611128490

We may want to modify the logic here to enforce that logic. Especially when the values of old and new labels do not match, we may want to return error.

Reply:
Well that should probably go into validation logic, rather than here...

Reply:
Well, it's on the NS, and we don't validate that, so I guess, we can't really except to just log and error and ignore?
/cc @tcnghia @mattmoor

Reply:
Logging the error and ignoring seems fine here, I don't know if we can do much better than that.